openshift/document-manager.dc.yaml

```diff
@@ -265,7 +265,7 @@ objects:
               key: DOCUMENT_MANAGER_CLIENT_SECRET
               name: "document-manager-secret"
         - name: DOCUMENT_MANAGER_URL
-          value: "https://document-manager${SUFFIX}.pharmanetenrolment.gov.bc.ca/"
+          value: "https://${VANITY_URL}/api/docman/"

```

Comment:
The document manager does not have a VANITY_URL parameter

Reply:
(hence why the URL we are seeing in file uploads is https://${VANITY_URL}/api ...)

Reply:
Correct, it needed to be added to the pipeline.  This is done.